gnomad_qc/v2/README.md

```diff
@@ -26,8 +27,19 @@ The scripts below are run approximately in the order they are listed.
 **annotations/**
 * `generate_qc_annotations.py` — compute and collate all variant- and allele-level annotations used for variant QC, including VEP; allele type; variant type; call stats (AC, AN, AF, homozygote count) for high-quality, release, and all samples; histograms of GQ, DP, and AB  metric distributions; and trio statistics (Mendelian errors and transmission disequilibrium test results)
 * `generate_frequency_data.py` — compute frequencies of variants in gnomAD for various sample groupings (e.g., ancestry, sex, and subpopulation) and for downsampled sets; add filtering allele frequency annotations for ancestry sample groupings
+* `generate_ld_data.py` - generate an linkage disequilibrium (LD) pruned set of variants using Hail's [ld_prune](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.ld_prune) method; calculate LD matrices by population using Hail's [ld_matrix](https://hail.is/docs/0.2/methods/genetics.html#hail.methods.ld_matrix) method; calculate LD scores for each population from the LD matrices; calculate cross-pop LD scores from the LD matricies; compute LD between SNVs and SVs
 
 **variant_qc/**
 * `correct_fafs.py` — temporary fix to address 2.1 bug; filter allele frequency annotations so that populations in which AC=1 have filtering allele frequencies set to 0
-* `prepare_data_release.py` — combine frequency, filtering alelle frequency, variant QC, VEP, dbSNP, and variant QC metric histogram annotations into unified table; add frequency annotations for each sample subset (e.g., controls, non-neuro, non-cancer, non-TOPMED); select and reformat release annotations for VCF export; create VCF header text; perform sanity check on release annotations
 * `make_var_annot_hists.py` — aggregate background distributions of specified variant QC metrics into histograms; first-pass run determines minimum and maximum values per metric; second-pass run aggregates QUAL metrics binned by allele frequency and aggregates other metrics over manually set bins/ranges, writes results out to json files (primarily for browser loading purposes)
+* `prepare_data_release.py` — combine frequency, filtering alelle frequency, variant QC, VEP, dbSNP, and variant QC metric histogram annotations into unified table; add frequency annotations for each sample subset (e.g., controls, non-neuro, non-cancer, non-TOPMED); select and reformat release annotations for VCF export; create VCF header text; perform sanity check on release annotations
+* `exomes_genomes_coverage.py` needs a description and probably should be moved as well. It looks like it primarily creates coverage files for the flagship LOF paper, but maybe we also used these for the browser?
```

Comment:
Needs an official description -- looks like this was just copied from my comments

Reply:
ugh, sorry, I copied what you wrote for each and changed, but missed this one.

Reply:
@nawatts is exomes_genomes_coverage.py used for the browser?

Reply:
> is exomes_genomes_coverage.py used for the browser?

No. All the scripts used to load the public release files into the browser are in the gnomad-browser repository.

Reply:
Ok, so since all of the output from this script goes to (what was) `gs://gnomad-public/papers/2019-flagship-lof` it sounds like this isn't really part of the gnomAD QC production. @gtiao do you think I should just add a note (as well as to this README) to the top of this script that this was used in the flagship LOF paper with a link and not as part of production?

Reply:
Yup, sounds good to me -- thanks for investigating, Julia!